src/main/java/no/nav/fo/veilarbarena/utils/AuthorizationUtils.java

```diff
@@ -0,0 +1,44 @@
+package no.nav.fo.veilarbarena.utils;
+
+import lombok.extern.slf4j.Slf4j;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Base64;
+import java.util.Objects;
+
+import static java.lang.System.getProperty;
+import static no.nav.brukerdialog.tools.SecurityConstants.SYSTEMUSER_PASSWORD;
+import static no.nav.brukerdialog.tools.SecurityConstants.SYSTEMUSER_USERNAME;
+
+@Slf4j
+public class AuthorizationUtils {
+
+    private static final String AUTHENTICATE = "WWW-Authenticate";
+    private static final String BASIC_REALM = "BASIC realm=\"srvveilarbportefolje\"";
```

Comment:
Tenker vi heller burde bruke servicebrukeren til veilarbarena

Reply:
Litt for rask `ctrl-c` ``ctrl-v`